nodejs/eks/examples/migrate-nodegroups/README.md

```diff
@@ -0,0 +1,4 @@
+# examples/aws-ts-eks-migrate-nodegroups
```

Comment:
There are a *ton* of moving parts in this testcase!  I suppose that's partly intentional to validate a complex real-world case, but I wonder if it makes sense to do something complex like this in `examples` and/or a blog, but then keep a more focused test case in this repo?  I worry that there are just a lot of ways this test may be brittle and/or difficult to maintain.  The complexity of the test seems to somewhat obscure the core of what the test case is really trying to test.

Reply:
You're right, this test is complex given the delicate migration scenario. I started this in the `examples` repo and then moved it here as I felt we would benefit from having this be actively tested, given that we have already gone through the work to script it.

What complexity sticks out the most to you? I aimed at making this use-case and the steps included to be as minimal as possible, and worked to hone that in over the last few weeks, but this flow is the reality of such an over-arching operation in k8s. Any ideas for alternative approaches?

Reply:
It wasn’t clear why the scenario required more than new cluster, new nodegroup, new workload.  Instead it has nearly a dozen ts files worth of infrastructure. 

At least as far as testing this here - it feels like we aren’t fundamentally gaining any additional test coverage from the additional complexity.

Reply:
Tests have been adapted. See [comments](https://github.com/pulumi/pulumi-eks/pull/195#issuecomment-511995592).